rawhttp-core/src/main/java/rawhttp/core/body/BodyReader.java

```diff
@@ -1,19 +1,16 @@
 package rawhttp.core.body;
 
-import java.io.ByteArrayOutputStream;
-import java.io.Closeable;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.nio.charset.Charset;
-import java.util.Iterator;
-import java.util.Optional;
-import java.util.OptionalLong;
```

Comment:
I will revert import changes later! My IDE lost my import settings apparently.... I avoid `*` imports as I've had many bugs caused by the wrong classes being imported accidentally.